app/controllers/emails_controller.rb

```diff
@@ -54,15 +54,26 @@ def save_template
       flash.now[:alert] = t('.saving_failed')
     end
     @event = Event.find(params[:event_id])
-    @templates = EmailTemplate.with_status(get_status)
+    @templates = EmailTemplate.with_status(get_email_template_status)
 
     render :email
   end
 
-  def get_status
+  def get_email_template_status
     params[:status] ? params[:status].to_sym : :default
   end
 
+  def get_corresponding_application_letter_status
+    case params[:status]
+    when "acceptance"
+      return :accepted
+    when "rejection"
+      return :rejected
+    else
+      return :accepted
+    end
+  end
```

Comment:
Just as little improvement. I think the case statement could also be written that way:
```ruby
return case params[:status]
       when "acceptance" then :accepted
       when "rejection" then :rejected
       else :accepted
       end
```